galleries/plot_types/stats/README.rst

```diff
@@ -1,6 +1,7 @@
 .. _stats_plots:
 
-Statistics plots
-----------------
+Statistical distributions
+-------------------------
 
-Plots for statistical analysis.
+Plots of the statistical distribution of a dataset ``X``. Some of these methods
+will also compute the distributions.
```

Comment:
```suggestion
Plots of statistical distributions. Some of these methods will also compute the distributions.
```
``X`` is not used elsewhere except for the boxplot parameter. It also does not convey structure. Therefore, it does not carry any information.